minibear/src/main/java/frc/robot/NavX/RegisterIO.java

```diff
@@ -0,0 +1,255 @@
+package frc.robot.NavX;
```

Comment:
similar for all files in this directory, the author here is unknown ... we should at least attribute who we think is the author